tests/helper/utils.py

```diff
@@ -5,6 +5,22 @@
 import subprocess
 
 
+def read_file_remote(client, file, remove_comments=False)-> list:
+    """ Read a file from the remote file system and returns it for
+        proper processing trunecated.
+    """
+    status, output = execute_remote_command(client, f"cat {file}", skip_error=True)
+    #assert status == 0, f"Error reading {file}"
```

Comment:
If functionality of removing comments or empty files is needed this could be a separate function.